extensions/sonarcloud/tasks/prepare/v1/task.json

```diff
@@ -11,8 +11,8 @@
   "author": "sonarsource",
   "version": {
     "Major": 1,
-    "Minor": 23,
-    "Patch": 1
+    "Minor": 24,
+    "Patch": 0
```

Comment:
if the S4NET is a patch version, should this be a patch version as well?

Reply:
applies to all version increments

Reply:
Not the same lifecycle, patch applies only for the code of the extension itself here. As long as we have a new version of the scanner, we bump the minor one.